internal/http/appeal.go

```
@@ -0,0 +1,454 @@
+		_ = appealStartTmpl.Execute(c.Writer, gin.H{
+			"Title":    "Appeal submitted",
+			"Subtitle": "Your appeal has been received.",
+			"Hash":     ticket.Hash,
+			"Done":     true,
+		})
+		return
+	}
+
+	// GET: show the appeal form or status
+	if ticket.Resolved {
+		status := "closed"
```

golangci-lint (2.12.2)

[error] ineffectual assignment to status

(ineffassign)